xml/MAIN-SBP-AMD-EPYC-4-SLES15SP4.xml

```diff
@@ -2926,6 +2926,11 @@ dmesg |grep SEV
     <para>For more information, refer to:</para>
 
     <itemizedlist>
+      <listitem>
+        <para>AMD EPYC 9004 Series Architecture Overview (<link
+            xlink:href="https://www.amd.com/system/files/documents/58015-epyc-9004-tg-architecture-overview.pdf"
+            >https://www.amd.com/system/files/documents/58015-epyc-9004-tg-architecture-overview.pdf</link>)</para>
```

Comment:
You can use an empty `<link/>` here as the text will be automatically generated from the link. No need to repeat yourself. :wink: 

```suggestion
            xlink:href="https://www.amd.com/system/files/documents/58015-epyc-9004-tg-architecture-overview.pdf"
            />)</para>
```